index.html

```diff
@@ -1,85 +0,0 @@
-        <meta content="width=device-width, initial-scale=1" name="viewport" />
-        <!-- <script src="https://kit.fontawesome.com/2eff1323a3.js"></script> -->
-    </head>
-    <body>
-        <div class="page">
-            <!-- Heading of the Page -->
-            <h2>Hi, I'm Sourav Kumar</h2>
-            <!-- Heading of the Page Ends here -->
-
-            <!-- Portfolio Image Here -->
-            <img src="./Assets/IMG_20160728_16404234.jpg" alt="Sourav's picture" height="250" width="250" class="centerImage">
-            
-            
-            <!-- Bio Section Start here -->
-            <div class="panel">
-                <h3 class="panelHead">Bio</h3>
-                <div class="panelBody">
-                    <p>I am a graduate in Electrical Engineering. I learned Full Stack development and have thorough knowledge of the latest technologies like 
-                    NodeJS, VueJS, ReactJS, and MongoDB. I can help you with web development or for web scrapping. You 
-                    can contact me Here.
-                    </p>
-                </div>
-            </div>
-            <!-- Bio Section Ends Here -->
-
-
-
-            <!-- Skills Section Starts Here -->
-            <div class="panel">
-                <h3 class="panelHead">Skills</h3>
-                <div class="panelBody">
-                    <p>I have following skills in my portfolio:
-                        <ul>
-                            <li>HTML</li>
-                            <li>CSS</li>
-                            <li>JavaScript</li>
-                            <li>NodeJS</li>
-                            <li>MongoDB</li>
-                            <li>VueJS</li>
-                            <li>ExpressJS</li>
-                            <li>Python</li>
-                            <li>Django</li>
-                            <li>Git</li>
-                        </ul>
-                    </p>
-                </div>
-            </div>
-            <!-- Skills Section Ends Here -->
-
-
-            <!-- Projects Section Starts Here -->
-            <div class="panel">
-                <h3 class="panelHead">Projects</h3>
-                <div class="panelBody">
-                    <p>The projects that I have done are given below:
-                        <ul>
-                            <li>Web Scrapper using NodeJS</li>
-                            <li>A To-Do List using NodeJS</li>
-                            <li>Much More Coming Soon...</li>
-                        </ul>
-                    </p>
-                </div>
-            </div>
-            <!-- Projects Section Ends Here -->
-
-
-            <!-- Contact Me section starts here -->
-            <div class="panel">
-                <h3 class="panelHead">Contact Me</h3>
-                <div class="panelBody">
-                    <p>Contact Here: Coming Soon</p>
-                </div>
-            </div>
-        </div>
-    </body>

```

Comment:
Why did you remove all of the code at once?
And do we have a server copy of this?